src/Tickets/Commerce/Order_Modifiers/Repositories/Order_Modifiers.php

```diff
@@ -203,38 +205,8 @@ public function find_by_id( int $id ): Order_Modifier {
 	 * @return Order_Modifier[] An array of Order_Modifiers or an empty array if none found.
 	 */
 	public function search_modifiers( array $params = [] ): array {
-		// Merge passed arguments with defaults.
-		$params     = wp_parse_args( $params, $this->get_default_query_params() );
-		$valid_args = $this->get_valid_params( $params );
-
-		// Start building the query.
-		$query = $this->get_query_builder_with_from();
-
-		// Add search functionality (search in display_name or slug).
-		if ( ! empty( $valid_args['search_term'] ) ) {
-			$query->whereLike( 'display_name', $valid_args['search_term'] );
-		}
-
-		// Set the order and orderby parameters.
-		if ( ! empty( $valid_args['order'] ) ) {
-			$orderby = array_key_exists( 'orderby', $valid_args ) ? $valid_args['orderby'] : 'display_name';
-			$query->orderBy( $orderby, $valid_args['order'] );
-		}
-
-		// Add the query limit.
-		if ( ! empty( $valid_args['limit'] ) ) {
-			$query->limit( $valid_args['limit'] );
-		}
-
-		// Add the query offset.
-		if ( ! empty( $valid_args['offset'] ) ) {
-			$query->offset( $valid_args['offset'] );
-		}
-
-		// Set the modifier type.
-		$query = $query->where( 'modifier_type', $this->modifier_type );
-
-		return $query->getAll() ?? [];
+		_deprecated_function( __METHOD__, 'TBD', 'get_modifiers' );
```

Comment:
Same as above, this was added in this release so we don't need to deprecate; we can just remove it.